packages/block-editor/src/hooks/layout-child.js

```diff
@@ -31,6 +31,16 @@ function useBlockPropsChildLayoutStyles( { style } ) {
 				flex-grow: 1;
 			}`;
 		}
+		if ( columnSpan ) {
+			css = `${ selector } {
```

Comment:
Should this be `+=`? Or should it be an `else if`?

Reply:
Hmmm yeah that's a good question. `selfStretch` and `flexSize` are only relevant to children of Row/Stack/flex blocks in general, whereas `columnSpan` is only relevant to grid children. We could potentially check the parent layout before outputting any of these styles, or we could just output everything because flex rules won't apply to grid and vice-versa. Not sure which is best 😄

Reply:
Ok I addressed this in the latest commit by adding an `else if ` and making sure that when a child layout is set on a bock, any previous layout config is wiped.